typed-racket-lib/typed-racket/rep/free-variance.rkt

```diff
@@ -102,6 +102,12 @@
      (remove-frees (flip-variances inner) name)]))
 
 
+(define (make-covariant frees)
+  (combined-frees
```

Comment:
(Unless I'm missing something) I'm pretty sure this function `make-covariant` should be deleted and anywhere that uses it should instead _just_ be calculating the free variables of the respective fields w/o modifying the variance.